tools/codegen/config.yml

```diff
@@ -65,3 +65,16 @@ resources:
     schema:
       aliases:
         group_id: project_id
+
+  stream_privatelink_endpoint:
```

Comment:
curious if any manual follow up change were needed 😄

Reply:
I had to adjust the attributes from `Optional: true` to either `Required: true` or `Computed: true`. There were minimal manual adjustments needed